packit/upstream.py

```diff
@@ -454,7 +444,7 @@ def create_archive(self, version: str = None):
                     f"{dir_name}/",
                     "HEAD",
                 ]
-            run_command(archive_cmd, cwd=self.local_project.working_dir)
+            self.command_handler.run_command(archive_cmd, return_output=True).strip()
```

Comment:
Why `strip()` there? There is not an assignment.

Reply:
whoops, copy-paste error